backend/decompme/settings.py

```diff
@@ -221,6 +225,10 @@
 SENTRY_DSN = env("SENTRY_DSN", str)
 SENTRY_SAMPLE_RATE = env("SENTRY_SAMPLE_RATE", float)
 
+SESSION_COOKIE_AGE = env("SESSION_COOKIE_AGE", int)
```

Comment:
Can you add a large default for this in .env.example? It looks like the default of the library will be only 3600s

Reply:
👍; mirrored the default into .env